src/stdio.jl

```diff
@@ -52,6 +102,18 @@ function watch_stream(rd::IO, name::AbstractString)
     end
 end
 
+send_stdout(t::Timer) = send_stream("stdout")
+send_stderr(t::Timer) = send_stream("stderr")
+
+function send_stream(name::AbstractString)
+    if bufs[name].size > 0
+        s = takebuf_string(bufs[name])
```

Comment:
Should we check for an invalid UTF8 string here (ala the `try`/`catch` in `read_stream`)?  If there is a partially encoded UTF8 character at the end of the buffer, which I suppose could theoretically happen although I've never seen it, it would be good to omit it from `s` and push those bytes back into the buffer (in the hope that the rest of the encoding will be pushed later).